tutorials/Python/07_FileHandling/FileHandling.ipynb

```diff
@@ -11,28 +11,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "One important aspect of using Python is being able to read and write files. To do this you use the *open* command:\n",
-    "*open*(*filename-in-quotes*, *mode*)\n",
+    "One important aspect of using Python is being able to read and write files. To do this you use the `open` command:\n",
     "\n",
-    "In place of *mode* you can specify:\n",
-    "- 'r', to open a file you want to read\n",
-    "- 'w', to open a file that you want to write (starting a new file).\n",
-    "- 'a', to open a file that you want to write (appending to an existing file).\n",
+    "```\n",
+    "open(filename-in-quotes, mode)\n",
+    "```\n",
     "\n",
-    "So, to open a file with filename file1.txt in 'read' mode:\n",
+    "In place of `mode` you can specify:\n",
+    "- `\"r\"`, to open a file you want to read\n",
+    "- `\"w\"`, to open a file that you want to write (starting a new file).\n",
+    "- `\"a\"`, to open a file that you want to write (appending to an existing file).\n",
     "\n",
-    "open('file1.txt', 'r')\n",
+    "So, to open a file with filename `file1.txt` in \"read\" mode:\n",
     "\n",
-    "To open a file, file2.txt, in 'write' mode:\n",
+    "```\n",
+    "open('file1.txt', \"r\")\n",
+    "```\n",
     "\n",
-    "open('file2.txt', 'w')\n",
-    "CAREFUL: this will create a new file called file2.txt.  If you had an existing file named file2.txt it will be overwritten.\n",
+    "To open a file, `file2.txt`, in \"write\" mode:\n",
     "\n",
-    "To open a file, file3.txt, in 'append' mode:\n",
+    "```\n",
+    "open('file2.txt', \"r\")\n",
+    "```\n",
     "\n",
-    "open('file3.txt', 'a')\n",
+    "CAREFUL: this will create a new file called `file2.txt`.  If you had an existing file named `file2.txt` it will be overwritten.\n",
     "\n",
-    "If you don't specify the mode, the default is 'r'.\n",
+    "To open a file, `file3.txt`, in 'append' mode:\n",
```

Comment:
Do you think it's worth explaining what append means?

Reply:
We are not using it later on... I think it's fine to just mention it; from the name is should be clear?